Picker/Picker.js

```diff
@@ -60,6 +60,20 @@ const PickerBase = kind({
 		 */
 		'aria-valuetext': PropTypes.string,
 
+		/**
+		 * Sets rules to determine the user interaction of the control
+		 * at a horizontal joined Picker case.
+		 *
+		 * When `enter`, the user use the enter key to change picker's value.
+		 * When `leftRight`, the user use the left/right keys to change the picker's value.
+		 * If [orientation]{@link sandstone/Picker.Picker.orientation} is 'vertical' or
+		 * [joined]{@link sandstone/Picker.Picker.joined} is not assigned or is false, this prop is ignored.
+		 *
+		 * @type {('enter'|'leftRight')}
+		 * @public
+		 */
+		changedBy: PropTypes.oneOf(['enter', 'leftRight']),
```

Comment:
I'm sorry but would you please change `leftRight` to `arrow`?

Reply:
changed it